src/android/index.jsx

```diff
@@ -260,7 +261,7 @@ class Android extends Component {
                         {
                           eq: {
                             framework: 10,
-                            repo: 'mozilla-central',
+                            repo: MOZILLA_CENTRAL.name,
```

Comment:
Please change the `filter` back to the original string

Reply:
I should change all the filter occurrences I changed back to the string format?

Reply:
Hi @Je-ni , sorry for the delay on this. It looks like Kyle gave an example of what he meant in this comment: 
https://github.com/mozilla-frontend-infra/firefox-health-dashboard/pull/667#discussion_r389386937